src/pages/billing/Project.js

```diff
@@ -525,14 +525,22 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     { ms: 5000 }
   )
 
-  const { displayName = null } = _.find({ accountName: billingProject.billingAccount }, billingAccounts) || { displayName: 'No Access' }
+  // (CA-1586) For some reason the api sometimes returns string null, and sometimes returns no field, and sometimes returns null. This is just to be complete.
+  const billingProjectHasBillingAccount = !(billingProject.billingAccount === 'null' || _.isNil(billingProject.billingAccount))
```

Comment:
This condition is taken from existing code that controls the disabled state of the "Remove Billing Account" button.